[BUG] UnboundLocalError: local variable 'subtitle_font_size' referenced before assignment

DOCKER部署0.3版本

[INFO] - [2025-04-15 09:15:24,471 bilive scan] - Begin processing /app/Videos/64458/64458_20250414-23-34-18.mp4...
[INFO] - [2025-04-15 09:15:24,680 bilive scan] - The video resolution x is 1280 and y is 720
[ERROR] - [2025-04-15 09:15:24,690 bilive scan] - TypeError: unsupported operand type(s) for *: 'NoneType' and 'int' - Check the return value of process_danmakus
[INFO] - [2025-04-15 09:15:24,925 bilive scan] - Generate subtitles via whisper api
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/local/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/app/src/burn/scan.py", line 75, in <module>
    process_folder_append(room_folder)
  File "/app/src/burn/scan.py", line 61, in process_folder_append
    render_video(file)
  File "/app/src/burn/render_video.py", line 75, in render_video
    original_video_path, format_video_path, subtitle_font_size, subtitle_margin_v
UnboundLocalError: local variable 'subtitle_font_size' referenced before assignment


![Image](https://github.com/user-attachments/assets/7d76f188-0a86-488c-a994-4b9b5bb0f2fd)
api已被调用

--- a/src/burn/render_video.py
+++ b/src/burn/render_video.py
@@ -54,17 +54,21 @@ def render_video(video_path):
     srt_path = original_video_path[:-4] + ".srt"
     jsonl_path = original_video_path[:-4] + ".jsonl"
 
-    # Recoginze the resolution of video
-    resolution_x, resolution_y = get_resolution(original_video_path)
     try:
+        # Recoginze the resolution of video
+        resolution_x, resolution_y = get_resolution(original_video_path)
         # Process the danmakus to ass and remove emojis
         subtitle_font_size, subtitle_margin_v = process_danmakus(
             xml_path, resolution_x, resolution_y
         )
-    except TypeError as e:
-        scan_log.error(f"TypeError: {e} - Check the return value of process_danmakus")
-    except FileNotFoundError as e:
-        scan_log.error(f"FileNotFoundError: {e} - Check if the file exists")
+    # except TypeError as e:
+    #     scan_log.error(f"TypeError: {e} - Check the return value of process_danmakus")
+    # except FileNotFoundError as e:
+    #     scan_log.error(f"FileNotFoundError: {e} - Check if the file exists")
+    except Exception as e:
+        scan_log.error(f"Error in process_danmakus: {e}")
+        subtitle_font_size = "16"
+        subtitle_margin_v = "60"
 
     # Generate the srt file via whisper model
     if MODEL_TYPE != "pipeline":

